src/libraries/System.Private.Xml/src/System/Xml/Schema/SchemaCollectionCompiler.cs

```diff
@@ -2073,7 +2073,7 @@ private void CompileAttribute(XmlSchemaAttribute xa)
                     else
                     {
                         decl.Presence = SchemaDeclBase.Use.Fixed;
-                        decl.DefaultValueRaw = decl.DefaultValueExpanded = xa.FixedValue;
+                        decl.DefaultValueRaw = decl.DefaultValueExpanded = xa.FixedValue!;
```

Comment:
Suppressed the nullability warning, this looks like nullability analysis bug, filed https://github.com/dotnet/roslyn/issues/70962, looks like the bug was preexisting. This XML code was not changed since it is added many years ago, have no idea why this is started failing now.